recipe/meta.yaml

```diff
@@ -51,6 +54,7 @@ outputs:
         - {{ stdlib("c") }}
         - make  # [unix]
         - cmake
+        - libprotobuf
```

Comment:
typically, upstream packages don't expect so much cross compilation. We often find ourselves forcing a given `protoc` for cross compilation:

https://github.com/conda-forge/grpc-cpp-feedstock/blob/2bd22985ba071f86130b12aee70c70b4f67fa4d5/recipe/build-cpp.sh#L46

Reply:
I have this cmake option:
![{4B2233BE-D389-4CF4-947C-7C7723AA79E9}](https://github.com/user-attachments/assets/7f55878f-410d-4533-9849-253383465dff)

not sure why it does not work

Reply:
My guess is that the other failure masked this change in the build system